composer.json

```diff
@@ -21,14 +21,14 @@
     ],
     "require": {
         "php": ">=5.6",
-        "symfony/symfony": "~2.8|~3.0",
+        "symfony/symfony": "~2.8|~3.0|^4.0-BETA",
```

Comment:
As well

Reply:
Thanks! I really love this project! 
forking it just to add 5 more characters in composer.json would somehow not be the ideal path to go ;-)

Reply:
Of course it isn't but as you can see yourself, in last 12 months or so, I'm the only one who really cares about this project. Sadly I don't have so much free time to dedicate to it